### PR TITLE
[Validator] Add missing validator translations in Polish language

### PR DESCRIPTION
**Pull request by stloyd**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Tickets       | n/a 
| License       | MIT
| Doc PR        | n/a 

Similar to https://github.com/symfony/symfony/pull/50590 but now in Polish language

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf [hunk 1]
```diff
@@ -402,6 +402,30 @@
...
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Ta wartość zawiera znaki które nie są dozwolone przez aktualny poziom ograniczeń.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Using invisible characters is not allowed.</source>
+                <target>Używanie niewidzialnych znaków jest niedozwolone.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Mixing numbers from different scripts is not allowed.</source>
+                <target>Mieszanie liczb z różnych skrytów jest niedozwolone.</target>
```
Comment (wkania) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: Typo:  skrytów -> skryptów

Comment (stloyd) [reply src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: Fixed thx.

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf [hunk 2]
```diff
@@ -402,6 +402,30 @@
...
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Nazwa pliku jest za długa. Powinna mieć {{ filename_max_length }} znak lub mniej.|Nazwa pliku jest za długa. Powinna mieć {{ filename_max_length }} znaków lub mniej.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>Siła hasła jest zbyt niska. Użyj mocniejszego hasła.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Ta wartość zawiera znaki które nie są dozwolone przez aktualny poziom ograniczeń.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]:
```suggestion
                <target>Ta wartość zawiera znaki, które nie są dozwolone przez aktualny poziom ograniczeń.</target>
```

**Review by GSadee — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @stloyd.

**Pull request merged**